### PR TITLE
Just test ci

### PR DESCRIPTION
**Pull request by AGaliuzov**
Please do not review do not merge do nont touch

### CODE REVIEW

**Comment by AGaliuzov** [comment]
Run CI

**Comment by AGaliuzov** [comment]
Run CI

**Comment by AGaliuzov** [comment]
Can one of the admins verify this patch?

**Comment by AGaliuzov** [comment]
Run CI

**Comment by AGaliuzov** [comment]
Run CI

**Comment by AGaliuzov** [comment]
Run CI

**Comment by AGaliuzov** [comment]
Run CI

**Comment by AGaliuzov** [comment]
Run CI

**Comment by AGaliuzov** [comment]
Run CI

**Comment by AGaliuzov** [comment]
Run CI

File: src/components/hmi_message_handler/test/include/hmi_message_handler/mock_subscriber.cc [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  dbus_connection_flush(conn_);
+  if (dbus_error_is_set(&err)) {
+	dbus_error_free(&err);
+    return false;
+  }
+  return true;
+}
+
+void MockSubscriber::Send(const std::string& message) {
+ // int a = message.length();
```
Comment (pestOO) [inline src/components/hmi_message_handler/test/include/hmi_message_handler/mock_subscriber.cc]: legacy?

File: src/components/hmi_message_handler/test/include/hmi_message_handler/mock_subscriber.cc [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+* LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+* CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+* SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+* INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+* CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+* ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+* POSSIBILITY OF SUCH DAMAGE.
+*/
+
+#include <dbus/dbus.h>
```
Comment (pestOO) [inline src/components/hmi_message_handler/test/include/hmi_message_handler/mock_subscriber.cc]: Why mocking class need additional external dependencies?

File: src/components/hmi_message_handler/test/CMakeLists.txt [hunk 3]
```diff
@@ -46,9 +46,10 @@ set(LIBRARIES
     ConfigProfile
 )
 
-set(SOURCES
-    ${COMPONENTS_DIR}/hmi_message_handler/test/hmi_message_handler_impl_test.cc
-)          
+set(SOURCES 
+${COMPONENTS_DIR}/hmi_message_handler/test/hmi_message_handler_impl_test.cc
```
Comment (pestOO) [inline src/components/hmi_message_handler/test/CMakeLists.txt]: ident?

**Comment by AGaliuzov** [comment]
Run CI

**Comment by AGaliuzov** [comment]
Run CI

**Comment by AGaliuzov** [comment]
@pestOO, please read PR description more carefully. This is test PR for CI

**Comment by AGaliuzov** [comment]
Run CI

**Comment by AGaliuzov** [comment]
Run CI

**Comment by AGaliuzov** [comment]
Run CI

**Pull request closed without merging**